module/mempool/herocache/backdata/heropool/pool.go

```diff
@@ -76,8 +92,16 @@ func NewHeroPool(sizeLimit uint32, ejectionMode EjectionMode, logger zerolog.Log
 	return l
 }
 
+// can be negative
```

Comment:
Please add a more comprehensive godoc for this method, what does it do, what is the expected input.

Reply:
ahh I think we had one in the previous version but while doing the rebase I lost it.
Well noticed !